src/pricecypher/datasets.py

```diff
@@ -100,23 +106,35 @@ def get_scope_values(self, dataset_id, scope_id, bc_id='all'):
             .scope_values(scope_id)
         )
 
-    def get_transaction_summary(self, dataset_id, bc_id='all'):
+    def get_transaction_summary(self, dataset_id, bc_id='all', intake_status=None):
```

Comment:
Would it be a good idea to use some kind of enum for the possible statuses that can be passed, or perform a check on the given value?

Reply:
Might be nice to add at some point, but currently I think this is fine. If you input invalid data, the dataset service will tell you 422: the given data was invalid (tho not what exactly was invalid)